app/bin/tools/set_package_withheld.dart

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) 2020, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:args/args.dart';
+import 'package:gcloud/db.dart';
+
+import 'package:pub_dev/account/backend.dart';
+import 'package:pub_dev/package/backend.dart';
+import 'package:pub_dev/package/models.dart';
+import 'package:pub_dev/service/entrypoint/tools.dart';
+import 'package:pub_dev/shared/datastore_helper.dart';
+
+final _argParser = ArgParser()
+  ..addOption('lookup', allowed: ['package', 'publisher', 'userid', 'email'])
+  ..addOption('update', allowed: ['true', 'false'])
+  ..addOption('reason', help: 'The reason of withheld status.')
+  ..addFlag('help', abbr: 'h', defaultsTo: false, help: 'Show help.');
+
+/// Sets the private key value.
+Future main(List<String> args) async {
+  final argv = _argParser.parse(args);
+  final withheldStatus = _parseValue(argv['update'] as String);
+  final withheldReason = argv['reason'] as String;
+  final lookupKey = argv['lookup'] as String;
+
+  if (argv['help'] as bool == true || lookupKey == null) {
+    print(
+        'Usage: dart set_package_withheld.dart --lookup package [pkg1] [pkg2] -- list status of packages');
+    print(
+        'Usage: dart set_package_withheld.dart --lookup publisher [publisher1] [publisher2] -- list status of packages from publishers');
+    print(
+        'Usage: dart set_package_withheld.dart --lookup userid [user1] [user2] -- list status of packages from uploaders');
+    print(
+        'Usage: dart set_package_withheld.dart --lookup email [email1] [email2] -- list status of packages from uploaders');
+    print(
+        'Usage: dart set_package_withheld.dart --update true --lookup email [email1] [email2] -- update status of packages from uploaders');
+    print(_argParser.usage);
+    return;
+  }
```

Comment:
This some obscure flags... can we make a confirm where it prints out what it's going to do... if `--update` is given...

Reply:
Updated, PTAL.